environment.yml

```diff
@@ -4,13 +4,14 @@ channels:
     - bioconda
     - defaults
 dependencies:
-    - black=21.10b0
+    - black=22.10
     - fastp>=0.23
     - fastqc>=0.11
     - gzip>=1.7
     - mash>=2.3
     - megahit>=1.2
     - pytest-cov>=3.0
+    - python>=3.9
```

Comment:
YEAT cannot install if the user's python version is < 3.9. Add this to allow users to upgrade if needed.

Reply:
Might want to add or update an entry in the change log describing why only Python >=3.9 is supported now.